frontend/src/language/no.json

```diff
@@ -112,5 +112,10 @@
     "Re-run full mission": "Kjør hele oppdraget på nytt",
     "Re-run failed and cancelled tasks in the mission": "Kjør feilet og kansellerte oppgaver i oppdraget på nytt",
     "Only active installations": "Bare installasjoner i bruk",
-    "Installation settings": "Installasjonsinstillinger"
+    "Installation settings": "Installasjonsinstillinger",
+    "Send robots to safe zone": "Sende roboter til sikker sone",
+    "Send robots to safe zone long text": "Du er i ferd med å sende robotene til en sikker sone og dermed stoppe alle pågående oppdrag. For å kjøre oppdragene igjen må robotene starte fra begynnelsen.",
+    "Send robots to safe confirmation text": "Er du sikker på at du vil sende alle robotene til en sikker sone?",
+    "Dismiss robots from safe zone long text": "Normal operation can resume by continuing the missions present on the queue or remove them all.",
```

Comment:
Translate this no norwegian

Reply:
I've made an attempt at the translation, but I'm especially unsure about the last one